Jenkinsfile

```diff
@@ -4,6 +4,6 @@
 buildPlugin(configurations: [
   [ platform: "linux", jdk: "17" ],
   [ platform: "windows", jdk: "17" ],
-  [ platform: "linux", jdk: "17", jenkins: "2.479.1", javaLabel: 17 ],
-  [ platform: "windows", jdk: "17", jenkins: "2.479.1", javaLabel: 17 ]
+  [ platform: "linux", jdk: "17", jenkins: "2.479.3", javaLabel: 17 ],
```

Comment:
It is recommended to use the lastest minor in an LTS line. Right now, `2.479.3` is recommended as a baseline https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/

Reply:
In addition I'd adive to have also a line of tests use JDK 21. See the recommendation from the Jenkins plugin `archetypes`: https://github.com/jenkinsci/archetypes/blob/master/common-files/Jenkinsfile